guide/using-the-e-puck-robot.md

```diff
@@ -111,12 +113,12 @@ The `wb_motor_set_velocity` and `wb_position_sensor_get_value` functions allow y
 %end
 
 The forward direction of the e-puck is given by the negative *z*-axis of the world coordinates.
-This is also the direction the eye of the camera is looking to; in keeping with the VRML97 standard, the direction vector of the camera is pointing in the opposite direction, namely the direction of the positive *z*-axis.
+This is also the direction in which the camera eye is looking; to comply to the VRML97 standard, the direction vector of the camera is pointing in the opposite direction, namely the direction of the positive *z*-axis.
```

Comment:
Minor: I'm wondering if explaining the reason of the axis choice is important enough to document it.

Reply:
Maybe @omichel has an idea

Reply:
I would simply remove "to comply with the VRML97 standard, ".